packages/config/src/VitessceAutoConfig.js

```diff
@@ -427,19 +581,38 @@ export async function generateConfigs(fileUrls) {
 
   const allViews = [];
 
+  const dataset = vc.addDataset(`${hintsConfig.title} dataset.`);
+
   fileUrls.forEach((url) => {
-    allViews.push(generateConfig(url, vc));
+    allViews.push(generateConfig(url, vc, dataset, hintsConfig, hintsType, useHints));
```

Comment:
Based on this line, it seems like `generateConfig` is not doing what its name would suggest - can we rename it to `generateViewDefinition`?